main/lib/src/TransportBase.cc

```diff
@@ -24,6 +25,16 @@ namespace eudaq {
     return true;
   }
 
+  int ConnectionInfo::GetRemoteInfo() const
```

Comment:
I think GetRemoteInfo return the port number by converting the m_host. 
GetRemoteInfo is only called from new intruduced function  `bool operator < (const ConnectionInfo comp) const` of TransportBase. 
Does the operator< used somewhere?  Otherwise, remove both of them since they are not used.